### PR TITLE
Fix environmental variable expansion

### LINKED ISSUE #1211: Commands Runner variables
Hi everyone
I want to make a discord webhook with a python script , but on the command runner 
python3 /home/defidegue/delete.py --user $USERNAME --name $FILE
Only the message has been sent , the variables "$USERNAME" or "$FILE" is not changed
I have also an 500 internal error when i delete a file
image
I'm on the 2.11.0 version
The command say blocked but is working
image

2020/12/29 04:55:41 [INFO] Blocking Command: "python3 /home/defidegue/delete.py --user $USERNAME --name $FILE"
@@ Notification sended
Thanks you !

### PR DESCRIPTION
**Pull request by WeidiDeng**
**Description**
Fix [1211](https://github.com/filebrowser/filebrowser/issues/1211).

:rotating_light: Before submitting your PR, please read [community](https://github.com/filebrowser/community), and indicate which issues (in any of the repos) are either fixed or closed by this PR. See [GitHub Help: Closing issues using keywords](https://help.github.com/articles/closing-issues-via-commit-messages/).

- [ ] DO make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [ ] DO make sure you are making a pull request against the **master branch** (left side). Also you should start *your branch* off *our master*.
- [ ] DO make sure that File Browser can be successfully built. See [builds](https://github.com/filebrowser/community/blob/master/builds.md) and [development](https://github.com/filebrowser/community/blob/master/development.md).
- [ ] DO make sure that related issues are opened in other repositories. I.e., the frontend, caddy plugins or the web page need to be updated accordingly.
- [ ] AVOID breaking the continuous integration build.

**Further comments**
If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did, what alternatives you considered, etc.

:heart: Thank you!

### CODE REVIEW

File: runner/runner.go [hunk 1]
```diff
@@ -65,6 +65,30 @@ func (r *Runner) exec(raw, evt, path, dst string, user *users.User) error {
...
+		case "SCOPE":
+			return user.Scope
+		case "TRIGGER":
+			return evt
+		case "USERNAME":
+			return user.Username
+		case "DESTINATION":
+			return dst
+		default:
+			return os.Getenv(key)
```
Comment (o1egl) [inline runner/runner.go:81]: I think using global env is not safe. Let's use an empty string as a default value

**Review by o1egl — CHANGES_REQUESTED** [review]
Please remove search related commits from this pr.

**Comment by o1egl** [comment]
@WeidiDeng please remove the expansion of any env variable by default.

**Comment by WeidiDeng** [comment]
I found out that there is a global setting called ```Execute on shell``` that is supposed to do the envars expansion. Guess that's overlooked.

Also user executed command on the built-shell is affected by that setting too. I think documentation needs to be made more specific.

**Pull request merged**